Sources/RunCatLocalization/Resources/RunnerName.xcstrings

```diff
@@ -1648,6 +2092,12 @@
             "state" : "translated",
             "value" : "힘, 지혜, 용기"
           }
+        },
+        "zh-Hans" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "塞尔达传说"
+          }
```

Comment:
Due to copyright concerns, please avoid calling it "The Legend of Zelda" and translate it literally.